client/app/lib/onboarding.coffee

```diff
@@ -46,11 +54,15 @@ module.exports = class Onboarding
         throw new Error 'Missing mandatory `steps` parameter' unless steps
 
         @user = user
-        @steps = steps.map (step) =>
-            stepModel = new Step step
-            stepModel.onValidated () =>
-                @handleStepSubmitted()
-            return stepModel
+        @steps = steps
+            .reduce (activeSteps, step) =>
+                stepModel = new Step step
+                if stepModel.isActive user
+                    activeSteps.push stepModel
+                    stepModel.onValidated () =>
+                        @handleStepSubmitted()
```

Comment:
Hmm… we can declare in https://github.com/cozy/cozy-proxy/pull/307/files#diff-753b070baa8f3302572f4381d40ac8daR80 

```coffee
handleStepSubmitted: =>
    @goToNext()
```

But if I understand code properly, this method can be overriden in Step, so there's a risk to loose the binding, right?

If this, just take it as it :)